google/cloud/storage/client.h

```diff
@@ -95,6 +96,102 @@ class GetBucketMetadataRequest {
   WellKnownParameters well_known_parameters_;
 };
 
+class InsertObjectMediaRequest {
```

Comment:
Add a TODO for supporting the other modifiers/query params as listed at https://cloud.google.com/storage/docs/json_api/v1/objects/insert ?